apps/mobile/mobile-app/components/impact/impact-stat.tsx

```
@@ -0,0 +1,60 @@
+	onPress,
+}: ImpactStatProps) {
+	const CardWrapper = onPress ? Pressable : View
+
+	return (
+		<CardWrapper onPress={onPress} className="flex-1">
+			<Box
+				className={`${backgroundColor} rounded-xl p-3 shadow-sm border border-gray-100 items-center justify-between h-[130px]`}
+			>
+				{/* Icon */}
+				<Box
+					className={`rounded-full bg-white w-8 h-8 items-center justify-center shadow-sm`}
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)